frontend/src/app/main/ui/workspace/tokens/management/create/form.cljs

```diff
@@ -960,18 +969,20 @@ custom-input-token-value-props: Custom props passed to the custom-input-token-va
               :on-external-update-value on-external-update-value
               :token-resolve-result (when (seq token-resolve-result) token-resolve-result)}]
             [:> input-tokens-value*
-             {:label label
+             {:aria-label label
               :placeholder placeholder
               :default-value value
               :on-blur on-blur
+              :icon icon
               :on-change on-change
               :token-resolve-result (when (seq token-resolve-result) token-resolve-result)}])]))]))
 
 (mf/defc typography-reference-input*
   [{:keys [default-value on-blur on-update-value token-resolve-result]}]
   [:> input-tokens-value*
-   {:label "Reference"
-    :placeholder "Reference"
+   {:aria-label (tr "labels.reference")
+    :placeholder (tr "workspace.tokens.reference-composite")
+    :icon i/text-typography
```

Comment:
I'm not sure how it works internally, but it might make sense to pass the icon name instead of the HTML. Then, this component could render the icon internally using the 'icon*' component.